### PR TITLE
[10.0][ADD] sale_order_report_with_note

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Superseeds https://github.com/OCA/sale-reporting/pull/42.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
@aheficent could you check and review?

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
Both, headers and footers are duplicated in the report. And the view is a bit ugly:
image

**Comment by AaronHForgeFlow** [comment]
Trying to create footer text on runbot and got the following:

>   File "/home/odoo/build/OCA/sale-reporting/sale_order_report_with_note/models/sale.py", line 88, in set_condition_header
    self.text_condition2.text = self.note2
  File "/.repo_requirements/odoo/odoo/fields.py", line 930, in __set__
    record.ensure_one()
  File "/.repo_requirements/odoo/odoo/models.py", line 4844, in ensure_one
    raise ValueError("Expected singleton: %s" % self)
ValueError: Expected singleton: sale.condition_text()

Also cannot click on create and edit in header and footer conditions

**Comment by MiquelRForgeFlow** [comment]
Fixed all errors. Now it works fine!

File: sale_order_report_with_note/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+{
+    'name': 'Sales Order Report With Note',
+    'version': '10.0.1.0.0',
+    'category': 'Reports/Qweb',
+    'license': 'AGPL-3',
+    "author": "Camptocamp SA,"
+              "Eficent, "
+              "Serpent CS,"
+              "Odoo Community Association (OCA)",
+    'website': "https://odoo-community.org/",
```
Comment (bodedra) [inline sale_order_report_with_note/__manifest__.py:17]: shouldn't be https://github.com/OCA/sale-reporting ?

Comment (MiquelRForgeFlow) [reply sale_order_report_with_note/__manifest__.py:17]: yes

**Comment by MiquelRForgeFlow** [comment]
But wasted work. Unfortunately, this PR should stall and not be merged. The module [sale_comment_template](https://github.com/OCA/sale-reporting/tree/10.0/sale_comment_template) already has the same features.

**Comment by MiquelRForgeFlow** [comment]
Features of this module moved to https://github.com/OCA/sale-reporting/pull/52.

**Comment by AaronHForgeFlow** [comment]
I think this can be closed @mreficent

**Comment by MiquelRForgeFlow** [comment]
Right

**Pull request closed without merging**